eng/CredScanSuppression.json

```diff
@@ -57,6 +47,36 @@
                 "sdk/keyvault/microsoft-azure-keyvault-cryptography/src/test/resources/secp256keynew.pem"
             ],
             "_justification": "File contains private key used by test code."
+        },
+        {
+            "file": [
+                "sdk/communication/azure-communication-common-perf/src/main/java/com.azure.communication.common.perf/FakeCredentialInTest.java",
+                "sdk/keyvault/azure-security-keyvault-certificates/src/test/java/com/azure/security/keyvault/certificates/FakeCredentialInTest.java",
+                "sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/FakeCredentialInTest.java",
+                "sdk/spring/spring-cloud-azure-service/src/test/java/com/azure/spring/cloud/service/implementation/storage/FakeCredentialInTest.java",
+                "sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/FakeCredentialInTest.java",
+                "sdk/storage/azure-storage-common/src/test/java/com/azure/storage/common/FakeCredentialInTest.java",
+                "sdk/storage/microsoft-azure-storage-blob/src/test/java/com/microsoft/azure/storage/FakeCredentialInTest.groovy"
+            ],
+            "_justification": "File contains fake key used by test code."
+        },
+        {
+            "file": [
+                "sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/implementation/ConfigurationClientCredentials.java"
+            ],
+            "_justification": "File contains keyword 'secret='."
+        },
+        {
+            "file": [
+                "sdk/core/azure-core/src/main/java/com/azure/core/http/ProxyOptions.java"
+            ],
+            "_justification": "File contains User Login Credentials."
+        },
+        {
+            "file": [
+              "sdk/spring/spring-cloud-azure-integration-tests/test-resources/jdbc/mysql/test-resources.json"
```

Comment:
We should not exclude an entire file like this unless we know all it contains is fake secrets.